pkg/tests/apis/provisioning/provisioning_test.go

```diff
@@ -648,7 +652,9 @@ func TestProvisioning_ExportUnifiedToRepository(t *testing.T) {
 	for _, test := range []props{
 		{title: "Test dashboard. Created at v0", apiVersion: "dashboard.grafana.app/v0alpha1", name: "test-v0"},
 		{title: "Test dashboard. Created at v1", apiVersion: "dashboard.grafana.app/v1beta1", name: "test-v1"},
-		{title: "Test dashboard. Created at v2", apiVersion: "dashboard.grafana.app/v2alpha1", name: "test-v2"},
+		{title: "Test dashboard. Created at v2alpha1", apiVersion: "dashboard.grafana.app/v2alpha1", name: "test-v2alpha1"},
+		// TODO: fix this test
+		// {title: "Test dashboard. Created at v2alpha2", apiVersion: "dashboard.grafana.app/v2alpha2", name: "test-v2alpha2"},
```

Comment:
@radiohead I'm struggling to get this test working. When debugging it I can see the v2alpha2 dashboard being created in the temp directory. However, it's named `test-v2alpha2.json` instead of `test-dashboard-created-at-v2alpha2.json`. Moreover, the json in this file has`"apiVersion": "dashboard.grafana.app/v2alpha1",` which is incorrect.

I've tried debugging this, but wasn't able to finde the root cause. Can I get some help from you? Or maybe @Proximyst - I can see you've been touching these files :)